packages/app/src/components/CardWithAttributes.tsx

```diff
@@ -39,6 +42,8 @@ export function CardWithAttributes({
   disclosedAttributes,
   disclosedPayload,
   disableNavigation = false,
+  isExpired = false,
```

Comment:
I nowhere see these passed? Revocation is still todo, but this one we already know right?

Also for revocation we probably need to add a last checked time as well?

Reply:
I've added the expired!

Reply:
What do you mean with the last check time? Would you want to show that in the UI?